Documentation/ceph-csi-snapshot.md

```diff
@@ -99,3 +100,78 @@ kubectl delete -f cluster/examples/kubernetes/ceph/csi/rbd/pvc-restore.yaml
 kubectl delete -f cluster/examples/kubernetes/ceph/csi/rbd/snapshot.yaml
 kubectl delete -f cluster/examples/kubernetes/ceph/csi/rbd/snapshotclass.yaml
 ```
+
+## CephFS Snapshots
+
+### SnapshotClass
+
+In [snapshotClass](https://github.com/rook/rook/tree/{{ branchName }}/cluster/examples/kubernetes/ceph/csi/cephfs/snapshotclass.yaml),
+the `csi.storage.k8s.io/snapshotter-secret-name` parameter should reference the
+name of the secret created for the cephfsplugin and `pool` to reflect the Ceph pool name.
+
+Update the value of the `clusterID` field to match the namespace that Rook is
```

Comment:
this can be explained only once we can move this to the top to keep common for both rbd and cephfs